### PR TITLE
ICA Cred CDK: Added ICA Service Token Cred for Staging

### PR DESCRIPTION
**Pull request by victorskl**

### CODE REVIEW

File: cdk/apps/ica_credentials/app.py [hunk 1]
```diff
@@ -36,6 +36,24 @@
...
 )
 
+# Staging
+IcaCredentialsDeployment(
+    app,
+    f"{CDK_APP_NAME}-stg",
+    "c9173925-a838-4394-9fc6-61cb93c252a1",
+    [
+        # staging_workflows
+        # FIXME
```
Comment (victorskl) [inline cdk/apps/ica_credentials/app.py:46]: @reisingerf Might also need to create `staging_workflows` project? @alexiswl Could you pls fill/update for FIXME here, please.

Comment (victorskl) [reply cdk/apps/ica_credentials/app.py:46]: Flo; also need fill master API key for Staging, pls. Please ditto README and, respective sec name in SM Console. Thanks.

Comment (alexiswl) [reply cdk/apps/ica_credentials/app.py:46]: > Might also need to create staging_workflows project?
@victorskl no, workflows from both the development_workflows and production_workflows are available in the staging project

File: cdk/apps/ica_credentials/secrets/runtime/notify_slack/notify_slack.py [hunk 2]
```diff
@@ -9,6 +9,8 @@
 def get_aws_account_name(id: str) -> str:
     if id == "472057503814":
         return "prod"
+    elif id == "455634345446":
+        return "stg"
```
Comment (andrewpatto) [inline cdk/apps/ica_credentials/secrets/runtime/notify_slack/notify_slack.py:13]: are we going to be consistent with this throughout everywhere? (as in is this setting the precedent for all projects and how we shorten staging?)

File: cdk/apps/ica_credentials/app.py (hunk 1 above)
Comment (alexiswl) [reply cdk/apps/ica_credentials/app.py:46]: > In a sense, no tailored token needed for workflows to be appeared in Staging project context. So, empty list here would just work, I reckon?

Nope, the workflows token is used by the cwl-ica GitHub actions but that's it

Comment (victorskl) [reply cdk/apps/ica_credentials/app.py:46]: Ah. I need help explain, pls. 😅 Shall we have a quick call, pls.

Comment (victorskl) [reply cdk/apps/ica_credentials/app.py:46]: As discussed. Empty list here is ok for now; as Staging workflows are pointed from either DEV or PROD.

**Review by andrewpatto — APPROVED** [review]

**Comment by victorskl** [comment]
Merging...

**Pull request merged**